src/Uno.UI.FluentTheme/Resources/PriorityDefault/FlipView_themeresources.xaml

```diff
@@ -95,7 +97,7 @@
         <Setter Property="ItemsPanel">
             <Setter.Value>
                 <ItemsPanelTemplate>
-                    <VirtualizingStackPanel AreScrollSnapPointsRegular="True" Orientation="Horizontal" />
+                    <StackPanel AreScrollSnapPointsRegular="True" Orientation="Horizontal" />
```

Comment:
Add a comment here marked 'Uno workaround' and explaining why this is needed, otherwise this will get lost when we update to newer versions of the MUX resources

Reply:
Done